src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -339,8 +357,31 @@
        :on-done (fn [{:keys [success? get-parsed-response]}]
                   (if success?
                     (do (swap! state assoc :is-leo-whitelisted? true)
-                        (this :-get-clusters-list-if-whitelisted))
+                        (this :-get-clusters-list-if-whitelisted)
+                        (this :-schedule-cookie-refresh-if-whitelisted))
                     (swap! state assoc :server-response {:server-error (get-parsed-response false)})))}))
+
+   :-schedule-cookie-refresh-if-whitelisted
+   (fn [{:keys [props state locals this]}]
+     (let [{{:keys [clusters]} :server-response} @state]
+       (when (and (not (:dead? @locals)) (:is-leo-whitelisted? @state))
```

Comment:
I hate that you have to check for `:dead?`, but the alternative is not a pattern we have yet established. What I'd prefer (requires refactoring) is to cancel both the AJAX call and the timeout in `:component-will-unmount` so this would just not fire. I should probably file a bug to do that...